models/zendesk__ticket_metrics.sql

```diff
@@ -89,16 +90,16 @@ select
   ticket_comments.count_internal_comments as total_agent_replies,
   
   case when ticket_enriched.is_requester_active = true and ticket_enriched.requester_last_login_at is not null
-    then (({{ dbt_utils.datediff("ticket_enriched.requester_last_login_at", dbt_utils.current_timestamp(), 'second') }} /60))
+    then({{ dbt_utils.datediff("ticket_enriched.requester_last_login_at", dbt_utils.current_timestamp(), 'second') }} /60)
```

Comment:
```suggestion
    then ({{ dbt_utils.datediff("ticket_enriched.requester_last_login_at", dbt_utils.current_timestamp(), 'second') }} /60)
```
not sure if that's needed but 🤷

Reply:
I much prefer having our code be clean and consistent, so I love this and will commit the suggestions. Thanks!! 💯